specification/maps/data-plane/Creator/preview/2022-09-01-preview/routeset.json

```diff
@@ -0,0 +1,360 @@
+      "scopes": {
+        "https://atlas.microsoft.com/.default": "https://atlas.microsoft.com/.default"
+      }
+    },
+    "SharedKey": {
+      "type": "apiKey",
+      "description": "This is a shared key that is provisioned when you [Create an Azure Maps account](https://docs.microsoft.com/azure/azure-maps/quick-demo-map-app#create-an-azure-maps-account) in the Azure portal or using PowerShell, CLI, Azure SDKs, or REST API.\n\n With this key, any application can access all REST API. In other words, this key can be used as a master key in the account that they are issued in.\n\n For publicly exposed applications, our recommendation is to use the [confidential client applications](https://docs.microsoft.com/azure/azure-maps/authentication-best-practices#confidential-client-applications) approach to access Azure Maps REST APIs so your key can be securely stored.",
+      "name": "subscription-key",
+      "in": "query"
+    },
+    "SasToken": {
+      "type": "apiKey",
+      "description": "This is a shared access signature token is created from the List SAS operation on the [Azure Maps resource](https://aka.ms/amauth) through the Azure management plane via Azure portal, PowerShell, CLI, Azure SDKs, or REST APIs.\n\n With this token, any application is authorized to access with Azure role-based access controls and fine-grain control to the expiration, rate, and region(s) of use for the particular token. In other words, the SAS Token can be used to allow applications to control access in a more secured way than the shared key.\n\n For publicly exposed applications, our recommendation is to configure a specific list of allowed origins on the [Map account resource](https://aka.ms/amauth) to limit rendering abuse and regularly renew the SAS Token.",
+      "name": "SAS Token",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://atlas.microsoft.com/.default"
+      ]
+    },
+    {
+      "SharedKey": []
+    },
+    {
+      "SasToken": []
+    }
+  ],
+  "definitions": {
+    "FacilityDetails": {
+      "description": "The level identifiers in the facility.",
+      "type": "object",
+      "properties": {
+        "facilityId": {
+          "description": "the identifier of the structure this routeset was created from.",
+          "type": "string",
+          "readOnly": true
+        },
+        "levelOrdinals": {
+          "description": "the level identifiers within this facility",
+          "type": "array",
+          "items": {
+            "type": "number"
```

Comment:
It is integer or float? Ask because the description is "identifiers", and float seems not a good candidate.

Reply:
it is an integer identifying the floor.it value is required in the wayfind API to identify which floor the endpoints refer to. that said, thanks for bringing this up: I reworded some of the descriptions to improve clarity.